skbio/stats/tests/test_composition.py

```diff
@@ -0,0 +1,117 @@
+from __future__ import absolute_import, division, print_function
+
+# ----------------------------------------------------------------------------
+# Copyright (c) 2013--, scikit-bio development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+from unittest import TestCase, main
+
+import numpy as np
+from numpy.testing import assert_array_almost_equal
```

Comment:
Please use `import numpy.testing as npt` for consistency.

When starting a module it sometimes looks cleaner to import just the function you're using, but then modules grow and end up with infinite lists of names imported.